kse/src/org/kse/gui/dialogs/DVerifyCertificate.java

```diff
@@ -225,7 +269,8 @@ private void closeDialog() {
 	}
 
 	private void browsePressed() {
-		JFileChooser chooser = FileChooserFactory.getKeyStoreFileChooser();
+		JFileChooser chooser = FileChooserFactory.getCrlFileChooser();
+		chooser.setCurrentDirectory(CurrentDirectory.get());
 		chooser.setDialogTitle(res.getString("DVerifyCertificate.ChooseCACertificatesKeyStore.Title"));
```

Comment:
Hello @jgrateron, (FYI @kaikramer)

Thanks for this new functionality, here is a question:

If the function `browsePressed`  is now for **CRL**  with `getCrlFileChooser`, is it possible to change the 2 res. strings:
- `DVerifyCertificate.ChooseCACertificatesKeyStore.Title`
- `DVerifyCertificate.CaCertificatesKeyStoreFileChooser.button`

**to**

- `DVerifyCertificate.ChooseCRLFile.Title`
-  `DVerifyCertificate.ChooseCRLFile.button`

more appropriate _(especially for translation [I plan to translate the `Verify Cert GUI` in French])._

- Could you make a PR?
or would you like that I make one?

Regards,
Th.

Reply:
Thanks, fixed. Send a PR